apps/ai-content-generator/src/hooks/config/useSaveConfigHandler.ts

```diff
@@ -26,11 +28,30 @@ const useSaveConfigHandler = (
 
     const currentState = await sdk.app.getCurrentState();
 
+    let currentEditorInterface = {};
+
+    if (currentState !== null) {
+      const { EditorInterface } = currentState;
+      currentEditorInterface = EditorInterface;
+    }
```

Comment:
I feel this could read faster
```suggestion
    const currentEditorInterface = currentState?.EditorInterface || {};
```